static/searchPage.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <title>Ogene - Group Search</title>
     <script src="https://maps.googleapis.com/maps/api/js?key=APIkey"></script>
-    <script src="/js/searchPage.js"></script>
+    <script src="/socket.io/socket.io.js"></script>
```

Comment:
Are we hosting the socket.io script on our site? I don't see the library added to the commit.

Reply:
In a separate conversation, we concluded that this was fine.

From the Socket.io documentation: "By default, the Socket.IO server exposes a client bundle at ` /socket.io/socket.io.js.`" 

More information can be found here: https://socket.io/docs/client-installation/